app/src/resources/runs/useNotifyRunQuery.ts

```diff
@@ -24,7 +24,7 @@ export function useNotifyRunQuery<TError = Error>(
   useNotifyService({
     topic: `robot-server/runs/${runId}` as NotifyTopic,
     setRefetchUsingHTTP,
-    options: { ...options, enabled: options.enabled && runId != null },
+    options: { ...options, enabled: options?.enabled != null && runId != null },
```

Comment:
On L17, `options` will always be defined. 
```suggestion
    options: { ...options, enabled: options.enabled != null && runId != null },
```